# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nautilus-actions/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- nautilus-actions-1.4/plugin/nautilus-actions.c.vanilla	2006-09-10 18:17:51.000000000 +0200
+++ nautilus-actions-1.4/plugin/nautilus-actions.c	2006-09-10 18:18:09.000000000 +0200
@@ -44,7 +44,7 @@
 }
 
 #ifndef HAVE_NAUTILUS_MENU_PROVIDER_EMIT_ITEMS_UPDATED_SIGNAL
-static void nautilus_menu_provider_emit_items_updated_signal (NautilusMenuProvider *provider)
+void nautilus_menu_provider_emit_items_updated_signal (NautilusMenuProvider *provider)
 {
 	//--> fake function for backward compatibility
 	//-> do nothing
